### PR TITLE
fix: added redirect for NotFound page

### PR DESCRIPTION
**Pull request by varenechko**

### CODE REVIEW

**Review by laker-ls — APPROVED** [review]

**Review by RozaZhaisanova — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ const App: FC = () => {
           <Auth />
         </Route>
         <Route path="*">
+          <Redirect to="/404-page" />
```
Comment (veatik90) [inline src/App.tsx]: name it not-found, please

`<Route path="/not-found">
  <NotFound />
</Route>
<Route path="*">
  <Redirect to="/not-found" />
</Route>`

File: src/App.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ const App: FC = () => {
           <Auth />
         </Route>
         <Route path="*">
```
Comment (veatik90) [inline src/App.tsx:18]: <Route path="/not-found">
    <NotFound/>
  </Route>
 <Route path="*">
    <Redirect to="not-found" />
  </Route>

File: src/App.tsx (hunk 1 above)
Comment (veatik90) [reply src/App.tsx:19]: > name it not-found, please

<Route path="/not-found">
  <NotFound />
</Route>
<Route path="*">
  <Redirect to="/not-found" />
</Route>

Comment (veatik90) [reply src/App.tsx:19]: > > name it not-found, please

Comment (veatik90) [inline src/App.tsx:19]: <Route path="/not-found">
  <NotFound />
</Route>
<Route path="*">
  <Redirect to="/not-found" />
</Route>

Comment (veatik90) [reply src/App.tsx:19]: > name it not-found, please

<Route path="/not-found">
  <NotFound />
</Route>
<Route path="*">
  <Redirect to="/not-found" />
</Route>

**Pull request merged**